### PR TITLE
Feature biomassphenology

### PR DESCRIPTION
**Pull request by dschlaep**
Functionality to adjust mean monthly vegetation phenology according to site-specific climatic conditions (#145)

### CODE REVIEW

File: R/sw_Vegetation.R [hunk 1]
```diff
@@ -928,286 +620,661 @@ get_season_description_v2 <- function(x) {
...
+
+  # Normalize temperature values (on y-axis) into [0, 0.5]
+  temp_zero <- -min(ref_temp, target_temp)
+  tmp1 <- temp_zero + ref_temp
+  tmp2 <- temp_zero + target_temp
+  temp_2max <- max(tmp1, tmp2) / vals_std[2]
+
+  ref_temp_norm <- tmp1 / temp_2max
+  ref_temp_norm_mean <- mean(ref_temp_norm)  # nolint
+  target_temp_norm <- tmp2 / temp_2max  # nolint
```
Comment (kpalmqui) [inline R/sw_Vegetation.R:895]: Unclear what is meant here by `# nolint`

File: R/sw_Vegetation.R [hunk 2]
```diff
@@ -928,286 +620,661 @@ get_season_description_v2 <- function(x) {
...
+      # Success of detection of temperature curves intersections
+      good_tcurves <- sapply(
+        xcrosses_tcurves,
+        rSW2utils::has_uniroots,
+        tol = tol2
+      )
+
+      if (all(good_tcurves)) {
+        # Locate (all) x-axis values where line crosses
+        #   (iii) nomarlized vegetation curves
```
Comment (kpalmqui) [inline R/sw_Vegetation.R:1034]: normalized

File: R/sw_Vegetation.R (hunk 1 above)
Comment (dschlaep) [reply R/sw_Vegetation.R:895]: See https://github.com/jimhester/lintr#project-configuration
`# nolint` is the default "regex pattern for lines to exclude from linting" by the `lintr` R package

File: R/sw_Vegetation.R (hunk 2 above)
Comment (dschlaep) [reply R/sw_Vegetation.R]: Thanks for catching the typo! Fixed with da89ddd04f73f095698765cc54ad17af8b16c961

**Comment by dschlaep** [comment]
@kpalmqui Thanks for the review! I will merge this PR into master once the checks on travis and appveyor passed -- should take some minutes only. Then, we can finalize the dev branches on rSFSTEP2 and rSFSW2!

**Pull request merged**